rust/altrios-core/altrios-proc-macros/src/history_vec_derive.rs

```diff
@@ -21,7 +21,20 @@ pub(crate) fn history_vec_derive(input: TokenStream) -> TokenStream {
         .map(|f| {
             let ident = f.ident.as_ref().unwrap();
             let ty = &f.ty;
+            let doc_attrs = &f
```

Comment:
@nreinicke @kylecarow 

The changes in this file copy doc strings from `state` fields to corresponding fields in `history`.